apps/fluent-tester/package.json

```diff
@@ -39,15 +39,15 @@
     "@fluentui-react-native/themed-stylesheet": "^1.1.0",
     "@fluentui-react-native/win32-theme": ">=0.5.2 <1.0.0",
     "@fluentui/react-native": ">=0.21.13 <1.0.0",
-    "@office-iss/react-native-win32": "^0.62.6",
     "react-native-svg": "^11.0.0"
   },
   "devDependencies": {
+    "@office-iss/react-native-win32": "^0.63.7",
     "@react-native-community/eslint-config": "^1.1.0",
     "@rnx-kit/cli": "^0.0.3",
     "@types/jasmine": "3.5.10",
     "@types/react": "^16.9.34",
```

Comment:
Do we have new react typings?

Reply:
react typings went from 16.9.X to 17.X.  So I dont think so.